README.md

```diff
@@ -1,29 +1,286 @@
+  - Tests are tracked manually in `.qw/test_mapping.csv` where multiple issues are separated by `;`, in the following structure
+    test name | issue(s) targetted
+    -- | --
+    InputVerifierTests::testWeightIsNegative | 40
+    InputVerifierTests::testConversionOfWeightUnits | 4;23
+    acceptance_test_user_entry.docx | 2
+  - The development team may want to write a script that allows them to get the names of all tests run and ensure that automated tests listed in the csv still exist, bonus points if you make this an automated test that fails if a test doesn't exist
+  - Adding the manual verification test scripts to version control is ideal, but you may also list the name
+  - the QW CI task will fail if:
+    - No parent issue is given for a PR
+    - A test doesn't exist which is mapped to the parent issue
+    - The structure of the `test_mapping.csv` has been altered, or `;` has not been used to separate targeted issues
+    - A test name is duplicated in `test_mapping.csv`
+
+### Design validation
+
+- QW does not require design validation documentation to be added to the repository,
+  but you may add the test script.
+  Ideally this would be in a plain text format. We suggest creating a
+  `validation` directory and storing the validation information there
+- As with [Design outputs and design verification](#design-outputs-and-design-verification), update the `.qw/test_mapping.csv` with the test script(s) that fulfill the user need.
+- Create a pull request from github and follow the instructions on the template
+  ![](https://hackmd.io/_uploads/BkYpd7ikp.png)
+  - A validation links to a user need, so link this type of issue in the PR
+  - Assign the pull request to the team member who will authorise the validation.
+    Once they have validated it and signed any required paperwork,
+    they should approve the PR.
+
+### Non-QW items
+
+- If you'd like to create an issue that doesn't relate to the medical device then use the `Non QW item` issue template.
+  This tags the issue with `qw-ignore` so that QW does not parse the issue.
+  ![](https://hackmd.io/_uploads/SkWcEzQgp.png)
+- If you'd like to create a pull request that doesn't relate to the medical device then tag the pull request with `qw-ignore`.
+
+## Closing QW items when they are not resolved by a PR
+
+- There may be times where an QW-tracked issue is required to be closed not by a PR.
+- You may close the issue as either `completed` or `not planned` ![](https://hackmd.io/_uploads/Sy22Bi9x6.png)
+- Then please add the `qw` tag that matches the reason. The options for tags are:`qw-close-duplicate`, `qw-close-wont-fix`, `qw-close-duplicate`, , `qw-close-cannot-replicate`, `qw-close-not-a-defect`
+- [name=Stef] Alternatively we could have a manual closure reason in the issue template, and then that is required to be filled with free text as a reason?
```

Comment:
No, there's a reason why these four are different and need to be handled separately and have fixed meanings.
Feels to me that you want to be able to search for "cannot replicate" at some point to see if there's a trend emerging. "Duplicate" also has a meaning because you should say what it is a duplicate of. This is useful for communicating with the raiser of the defect. "won't fix" needs a risk assessment (once we are doing that). "Not a defect" probably just needs some free text. So we will probably at some point be producing separate reports for cannot reproduce and won't fix, and the duplicate/not a defect distinction is just for information for the raiser (unless we want to review the NADs, in which case we don't care about dupes).
Appropriately, you have duplicated `qw-close-duplicate`.